webmunk-ext/src/worker/Worker.ts

```diff
@@ -68,17 +69,39 @@ export class Worker {
     await chrome.storage.local.set({ surveys: this.surveys });
   }
 
+  private async saveParamsToStorage(params: Record<string, boolean>): Promise<void> {
+    await chrome.storage.local.set({ queryParams: params });
```

Comment:
Local storage property 'queryParams' is too general. Please rename to 'personalizationConfigs'.